socs/agents/acu/agent.py

```diff
@@ -160,6 +169,17 @@ def __init__(self, agent, acu_config='guess'):
                             self.clear_faults,
                             blocking=False)
 
+        # Automatic exercise program...
+        if exercise_plan:
+            agent.register_process(
+                'exercise', self.exercise, self._simple_process_stop)
+            self.agent.register_feed('activity',
+                                     record=True,
+                                     buffer_time=0,
+                                     agg_params={
+                                         'frame_length': 600,
```

Comment:
Just to point out this frame length is 10 times longer than other feeds in this agent register with.

Reply:
Added a note.